### ISSUE TITLE
Allow user to select desired layer colour on click

### ISSUE THREAD

**Issue by zeyadhazem** [DDMAL/Pixel.js#74]
The current code structure supports specifying red, green, blue and alpha values to layers. The only thing left is to create and link HTML UI elements to the code to support user input

**Comment by PotassiumK** [comment]
Still only one colour per layer though right?

**Comment by zeyadhazem** [comment]
Well I think it's up to the user to choose whatever layer colour they want, this won't affect internal functionality

**Comment by PotassiumK** [comment]
Yeah absolutely, I was just asking if you meant colour for the whole layer or actually adding user-side control of different colours on a same layer.
